spec/controllers/steps/additional_code_controller_spec.rb

```diff
@@ -35,7 +35,7 @@
       end
 
       it { expect { response }.to change(user_session, :additional_code_uk).from({}).to('105' => '2300') }
-      it { expect(response).to redirect_to(vat_path) }
+      it { expect(response).to redirect_to(excise_path('306')) }
```

Comment:
I updated the fixture to include an excise applicable additional code. We're testing the routing logic elsewhere. This just validates the redirect happens